main.go

```diff
@@ -95,7 +99,7 @@ func main() {
 	))
 
 	printVersion()
-
+	// todo: Should we not to do this deprecations now in SDK (ASAP) and then may we can just remove from here?
 	// Deprecated: --max-workers flag does not align well with the name of the option it configures on the controller
 	//   (MaxConcurrentReconciles). Flag `--max-concurrent-reconciles` should be used instead.
 	if pflag.Lookup("max-workers").Changed {
```

Comment:
Should we not to do this deprecations now in SDK (ASAP) and then may we can just remove from here?

Reply:
I need to think about this one more. I definitely see what you're getting at (i.e. let's get `--max-workers` removed before 1.0), but it's not super important for us to make this name change. We could just leave `--max-workers` and revert the switch to `--max-concurrent-reconciles`.